src/rules.js

```diff
@@ -21,7 +21,7 @@ const block = {
     + '|<\\?[\\s\\S]*?(?:\\?>\\n*|$)' // (3)
     + '|<![A-Z][\\s\\S]*?(?:>\\n*|$)' // (4)
     + '|<!\\[CDATA\\[[\\s\\S]*?(?:\\]\\]>\\n*|$)' // (5)
-    + '|</?(tag)(?: +|\\n|/?>)[\\s\\S]*?(?:\\n{2,}|$)' // (6)
+    + '|</?(tag)(?: +|\\n|/?>)[\\s\\S]*?(?:\\n *\\n+|$)' // (6)
```

Comment:
That regex is vulnerable because `\n` is matched by `\s*` and `\n+` after it.

We should probably do something like `[ \t]*` to include tabs.

Reply:
I fixed it to include tabs

Reply:
I think that tabs might not be needed because tabs are converted to spaces earlier on: 

https://github.com/markedjs/marked/blob/d4699f42cedaec1d2213b6c6d0773c56a7abc198/src/Lexer.js#L110

This was my impression from my debugging session some time ago but I might be wrong.

Reply:
@krassowski nice catch. I updated this PR again.